greedy_algorithms/kruskals_minimum_spanning_tree.cpp

```diff
@@ -15,7 +48,12 @@ void findMinimumEdge(int INFINITY, std::array<std::array<int, 6>, 6> graph) {
                   << std::endl;
```

Comment:
I've heard people saying using \n is faster what do you think?

Reply:
Same. Let's use that, it doesn't harm. 🙂

Reply:
The question about faster "depends" but `endl` causes the buffer to flush (so probably more allocations - but don't quote me on it, the compiler might already be optimising some stuff.... maybe) which means if you flush too much, say in a loop, then it would perform slower as compared to `\n` which won't flush the buffer. But if you're using it sparingly in only one or two places, and at the end of the stream anyways, then it might have little to no impact 😄